index.js

```diff
@@ -669,6 +1030,19 @@ module.exports.getWindowOpen = function(sid, ain, options)
     });
 };
 
+/* The above should be replaced by this, even though the whole device list is queried
```

Comment:
agreed, could you implement the replacement?

Reply:
Done